hw/ip/lc_ctrl/doc/_index.md

```diff
@@ -195,41 +200,220 @@ This ensures that during specific states (RAW, TEST_LOCKED, SCRAP, INVALID) it i
+The isolated partition is both read and writable during the DEV / PROD / PROD_END / RMA states.
+In all other states it is inaccessible, except during the TEST_UNLOCKED state where the partition is write-only.
+This construction allows to write a value to that partitition and keep it secret before advancing into any of the production states.
+
+
+## OTP Collateral
+
+The following is a list of all life cycle related collateral stored in OTP.
+Most collateral also contain associated metadata to indicate when the collateral is restricted from further software access, see [accessibility summary]({{< relref "#otp-accessibility-summary-and-impact-of-provision_en" >}}) for more details.
+Since not all collateral is consumed by the life cycle controller, the consuming agent is also shown.
+
+{{< snippet "hw/ip/lc_ctrl/doc/lc_ctrl_otp_collateral.md" >}}
+
+The TOKENs and KEYS are considered secret data and are stored in [wrapped format]({{< relref "#conditional-transitions">}}).
+Before use, the secrets are unwrapped.
+
+The SECRET0_DIGEST and SECRET2_DIGEST are the digest values computed over the secret partitions in OTP holding the tokens and root keys.
+As described in more detail in the [OTP controller specification]({{< relref "hw/ip/otp_ctrl/doc/_index.md#direct-access-memory-map">}}), these digests have a non-zero value once the partition has been provisioned and write/read access has been locked.
+
+### ID State of the Device
+
+If the SECRET2_DIGEST is zero, the device is considered to have "blank" ID state, in which case the CREATOR_ROOT_KEY_* (in OTP) and CREATOR_DIV_KEY (in FLASH) can be written by software.
+All consumers of these keys are supplied with an invalid value.
+
+If the SECRET2_DIGEST has a nonzero value, the device is considered "creator personalized", and the CREATOR_ROOT_KEY and CREATOR_DIV_KEY are no longer accessible to software.
+Actual values are supplied to the consumers.
+If SECRET2_DIGEST has a nonzero value, the CREATOR_SEED_SW_RW_EN signal will be disabled in PROD, PROD_END and RMA states.
+
+### Secret Collateral
+
+Among the OTP life cycle collateral, the following are considered secrets (note there may be other secrets unrelated to life cycle, please see [OTP controller specification]({{< relref "hw/ip/otp_ctrl/doc/_index.md#partition-listing-and-description">}}) for more details):
+
+- *_TOKEN
+- CREATOR_ROOT_KEY*
+
+Specifically this means after OTP sensing, the above entries are unwrapped to obtain the real value.
+Similarly, during programming, they are wrapped before beginning to be written to OTP.
+
+The function used for this wrapping is the lightweight PRESENT-cipher.
+The wrapping is a one time event during controlled manufacturing, and unwrapping also cannot be supplied with arbitrary ciphertexts.
+Thus the system cannot be abused to generate a large number of traces for informational leakage, and thus a fully hardened cipher (such as AES) is not required.
```

Comment:
i would say `masked-AES` just in case someone wants to argue that AES can also be unhardened.

Reply:
ok will fix